os/pm/pm_procfs.c

```diff
@@ -118,6 +118,7 @@ struct power_procfs_entry_s {
 
 /* Added to record timer countdown interrupt */
 struct pm_wakeup_timer_s g_timer_wakeup = { 0, 0 };
+struct pm_wakeup_timer_s g_timer_lock = { 0, 0 };
```

Comment:
Instead of maintaining multiple pm_wakeup_timer_s (as the development going, we might have more timer utilization scenarios). Will it be better to expand definition of pm_wakeup_timer_s, to handle more kinds of scenario?